tests/unit/test_notebooks_gpu.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-import shutil
+from tempfile import TemporaryDirectory
```

Comment:
for the tests, we could use the pytest fixture tmpdir https://docs.pytest.org/en/latest/tmpdir.html. It's quite neat. There is also tmpdir_factory available in case we need to have more control on the temporary folders

Reply:
Good! I'm working on it.